### ISSUE TITLE
How to change image name output in 'test_folder.py'

### ISSUE THREAD

**Issue by pydxflwb** [NVlabs/DG-Net#43]
Hello! Thanks for your excellent job!
I'm now trying to use DG-Net to create dataset. However I noticed that image-saving codes in 'test_folder.py' like this:

`im.save(dst_path + '/%03d_%03d_gan%s.jpg'%(label2, label, name[:-4]))`

` im.save(dst_path2 + '/%03d_%03d_gan%s.jpg'%(label2, label, name[:-4]))`
and 
`image_datasets = datasets.ImageFolder(opts.input_folder, data_transforms)`

`dataloader_content = torch.utils.data.DataLoader(image_datasets, batch_size=opts.batchsize, shuffle=False, pin_memory=True, num_workers=1)`

`dataloader_structure = torch.utils.data.DataLoader(image_datasets, batch_size=opts.batchsize, shuffle=True, pin_memory=True, num_workers=1)`

`image_paths = image_datasets.imgs`

So the name of pictures is , for example '001_010_gan0043_c3_xxxxxx.jpg'
Here 0043_c3 is the dataset label of label 010, which represents the structure provider.
I wonder if any provided code exists in the DG-Net code, to help me get the picture name of label 001.(appearance provider)
Or I have to modify my DataLoader to get names XD.
I'd appreciate it if you can give me some help.

**Comment by layumi** [comment]
Thanks @pydxflwb 
You may consider to modify the 'datasets.ImageFolder' (return one more variant, i.e., path.)
For example, you may take a look at ReIDFolder to build your own `Folder`.

https://github.com/NVlabs/DG-Net/blob/master/reIDfolder.py

**Comment by pydxflwb** [comment]
Thanks for your help. I'll take a try.
